specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/azureADOnlyAuthentication.json

```diff
@@ -0,0 +1,299 @@
+      "get": {
+        "operationId": "AzureADOnlyAuthentications_Get",
+        "summary": "Get Azure Active Directory only authentication property",
+        "description": "Gets a Azure Active Directory only authentication property",
+        "tags": [
+          "WorkspaceAzureADOnlyAuthentications"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/azureADOnlyAuthenticationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "schema": {
+              "$ref": "#/definitions/AzureADOnlyAuthentication"
+            },
+            "description": "OK"
+          },
+          "default": {
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            },
+            "description": ""
+          }
+        },
+        "consumes": [],
+        "produces": [
+          "application/json"
+        ],
+        "x-ms-examples": {
+          "Get Azure Active Directory Only Authentication property": {
+            "$ref": "./examples/GetAzureADOnlyAuthentication.json"
+          }
+        }
+      },
+      "put": {
+        "operationId": "AzureADOnlyAuthentications_Create",
+        "summary": "Create or Update Azure Active Directory only authentication property",
+        "description": "Create or Update a Azure Active Directory only authentication property for the workspaces",
+        "tags": [
+          "WorkspaceAzureADOnlyAuthentications"
+        ],
+        "deprecated": false,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/workspaceNameParameter"
+          },
+          {
+            "$ref": "./azureADOnlyAuthentication.json#/parameters/azureADOnlyAuthenticationNameParameter"
```

Comment:
>./azureADOnlyAuthentication.json [](http://example.com/codeflow?start=21&length=32)

If we reference something within the same file, we usually omit the file name

Reply:
+1

Reply:
Thanks, updated it to `AuthenticationState`

Reply:
Thanks, fixed in all places.